src/components/shared/MintButton.tsx

```diff
@@ -65,7 +65,7 @@ const MintButtonComponent: React.StatelessComponent<Props> = props => {
       }
     : undefined;
   return (
-    <React.Fragment>
+    <View>
```

Comment:
What's the difference?

Reply:
Grouping, if it's React.Fragment then spaced-evenly applies